static/js/user_events.js

```diff
@@ -68,7 +68,9 @@ exports.update_person = function update(person) {
 
     if (Object.prototype.hasOwnProperty.call(person, 'avatar_url')) {
         const url = person.avatar_url;
+        const avatar_version = person.avatar_version;
         person_obj.avatar_url = url;
+        person_obj.avatar_version = avatar_version;
```

Comment:
This can just be a single line: `person_obj.avatar_version = person.avatar_version`.

Reply:
hmm I thought there was a reason for this because we have `const url = person.avatar_ulr;` and then `person_obj.avatar_url = url;` but I saw the commit that introduced this and it doesn't seems to be a reason for that. Can I also change to `person_obj.avatar_url = url;` directly, @aero31aero ?

Reply:
For url, it looks like we have two occurences of it so someone must have thought that writing `const url = ...` once is better than writing the full variable name twice.

You could let it be as it is. Even your change is just a minor nitpick.

Reply:
Ok, thanks! :)